frontend/app/src/locales/en.json

```diff
@@ -214,6 +214,41 @@
     "hint": "Select as the main currency",
     "profit_currency": "Profit Currency"
   },
+  "date_format_help": {
+    "close": "Close",
+    "directive": {
+      "ampm": "morning or afternoon marker",
+      "day_numeric": "Day (numeric)",
+      "day_of_the_week": "Day of the week numeric",
+      "day_of_year": "Day of the year",
+      "day_of_year_padded": "Three-digit day of the year (with leading zeroes)",
+      "day_two_digit": "Two-digit day",
+      "hour": "Hour 24h",
+      "hour_padded": "Two-digit hour (with leading zeroes) 24h",
+      "hour_twelve": "Two-digit hour (with leading zeroes) 12h",
+      "hour_twelve_padded": "Hour 12h",
+      "locale_date": "Date in the locale format",
+      "locale_datetime": "Date and time in the locale format",
+      "locale_time": "Time in the locale format",
+      "minutes": "Minutes",
+      "minutes_padded": "Two-digit minutes (with leading zeroes)",
+      "month": "Full month name",
+      "month_numeric": "Month (numeric)",
+      "month_short": "Abbreviated month name",
+      "month_two_digit": "Two-digit month",
+      "seconds": "Seconds",
+      "seconds_padded": "Two-digit seconds (with leading zeroes)",
+      "timezone": "Time zone",
+      "timezone_offset": "RFC 822 numeric offset from GMT",
+      "week_day": "Full weekday name",
+      "week_day_short": "Abbreviated weekday name",
+      "year": "Four-digit year",
+      "year_two_digit": "Last two digits of the year"
+    },
+    "example": "e.g. ({example})",
```

Comment:
This is not really an example. It's what the format would translate to. So perhaps say "Translates to" or "Example result".

Reply:
For me, that would be an example

Reply:
All right. Please just write my suggestion though since the example would be a format string, not the result of the translation.

Reply:
![image](https://user-images.githubusercontent.com/2269732/118816280-4cf8df00-b8b2-11eb-8ba0-118146450057.png)

If that makes you happy sure I will do the change, though it doesn't make any sense to me. What is the practical difference between e.g. and `example result`? Why would the example be the `%A`

Reply:
https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

Reply:
![image](https://user-images.githubusercontent.com/2269732/118821905-f5f60880-b8b7-11eb-870d-41abca975979.png)